### DISCUSSION TITLE
Analyzing glycoproteomics data

### DISCUSSION THREAD

**Discussion by Dmorgen** [vdemichev/DiaNN#264] [Q&A]
Hi,

I'm interested in running DIA runs of glycoproteomics data based on EThcD data, analysed using Byonic.

Is it possible to run EThcD data using DIA-NN? since most of the fragments are glycan fragments - how should I provide this data in the library?

Thanks!
David.

**Comment by vdemichev** [comment]
Hi David,

Each fragment in a library, for it to be readable by DIA-NN, should be annotated as 'y' or 'b' fragment type. Even if the fragment is z-series, for example, it should be annotated as 'y'. So 'y' - any fragment that contains the C-terminus, 'b' - N-terminus. For oxonium fragments and anything else which does not contain a peptide terminus, I can just suggest to randomly write in either 'y' or 'b'. I don't know how reliable the FDR reported by DIA-NN will be if many peptides contain many such fragments. 

Best,
Vadim

**Reply by Dmorgen** [reply]
Thanks!
David.
